src/standardTypes.js

```diff
@@ -1,10 +1,15 @@
+const List = ({ args: [tpe] }, { gen }) => `t.list(${gen(tpe)})`;
+
 export default {
   Int: () => 't.Number',
   Float: () => 't.Number',
   String: () => 't.String',
   Boolean: () => 't.Boolean',
   Date: () => 't.String',
   DateTime: () => 't.String',
-  List: ({ args: [tpe] }, { gen }) => `t.list(${gen(tpe)})`,
-  Option: ({ args: [tpe] }, { gen }) => `t.maybe(${gen(tpe)})`
+  List,
+  Option: ({ args: [tpe] }, { gen }) => `t.maybe(${gen(tpe)})`,
+  Set: List,
+  TreeSet: List,
+  Map: ({ args: [tpe] }, { gen }) => `t.dict(t.String, ${gen(tpe)})`
```

Comment:
well no it could be an enum too in terms of tcomb, or a custom refinement over string, or maybe something more 👍